app/views/nonprofits/supporters/_listings.html.erb

```diff
@@ -41,12 +41,12 @@
     </td>
   </tr>
 	<tr>
-    <!--= show_if (any this.address this.city this.state_code this.zip_code this.country) -->
+    <!--= show_if (any this.primary_address.address this.primary_address.city this.primary_address.state_code this.primary_address.zip_code this.primary_address.country) -->
     <td><i class='fa fa-map-marker u-color--grey'></i></td>
     <td>
-      <!--= put address_with_commas this.address this.city this.state_code -->
-      <!--= put this.zip_code -->
-      <!--= put this.country -->
+      <!--= put address_with_commas this.primary_address.address this.primary_address.city this.primary_address.state_code -->
+      <!--= put this.primary_address.zip_code -->
+      <!--= put this.primary_address.country -->
```

Comment:
As I understand part 2, I think we're support to keep everything about Supporter object's "shape" the same but the address information would be coming from primary_address. Is that accurate?

If so, then this change shouldn't be needed.